pkg/proxy/expose.go

```diff
@@ -220,6 +220,7 @@ func exposeConfigMap(sso *apiv1.SSO, serviceName string) (*v1.ConfigMap, error)
 		HTTP:     false,
 		TLSAcme:  true,
 		Services: []string{serviceName},
+		UrlTemplate: sso.Spec.UrlTemplate,
```

Comment:
Please could you change this with a `string` in order to avoid coupling with the SSO CRD.

Reply:
I guess I don't understand the ask here.  Isn't it already bound because of sso.Spec.Domain?

Reply:
You're right. Sorry